src/client/Microsoft.Identity.Client/Platforms/Android/AndroidBrokerHelper.cs

```diff
@@ -405,7 +405,8 @@ private Intent GetInteractiveBrokerIntent(IDictionary<string, string> brokerPayl
                 ClientAppVersion = Application.Context.PackageManager.GetPackageInfo(Application.Context.PackageName, PackageInfoFlags.MatchAll).VersionName,
                 ClientVersion = GetValueFromBrokerPayload(brokerPayload, BrokerParameter.ClientVersion),
                 CorrelationId = GetValueFromBrokerPayload(brokerPayload, BrokerParameter.CorrelationId),
-                Prompt = "NONE"
+                Prompt = GetValueFromBrokerPayload(brokerPayload, BrokerParameter.Prompt).ToUpperInvariant(),
```

Comment:
Just found a bug in the broker where if you send the prompt without making the text uppercase the json parsing will sometimes fail on the broker side and crash the broker app. It just so happened that adding claims to the json trigged this bug. They are going to fix in the next release.